### DISCUSSION TITLE
Security Onion 2.3.70 now available!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#5203] [Announcements]
https://blog.securityonion.net/2021/08/security-onion-2370-now-available.html

![screenshot_1](https://user-images.githubusercontent.com/1659467/130054420-353956ff-f82f-4a20-a803-fd1f02fd6260.png)

**Comment by Git-Me-Some-Hub** [comment]
@dougburks  - I do not see anything listing the ECS compliance, did that get pushed from the .70 release?

**Reply by TOoSmOotH** [reply]
Zeek and Suricata ECS parsers are not in a place where we can use them. We have been working to make upstream changes to them but that is out of our control. I don't think we will see compliance this year on those parsers.

**Reply by Git-Me-Some-Hub** [reply]
@TOoSmOotH  understood, but if it is not for Zeek and Suricata does that mean they will be or are for the others like beats?
